modules/heat_conduction/test/tests/view_factors/view_factor_2d.i

```diff
@@ -0,0 +1,112 @@
+[GlobalParams]
+  view_factor_object_name = unobstructed_vf
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 2
+  ymin = 0
+  ymax = 1
+  nx = 2
+  ny = 2
+[]
+
+[UserObjects]
+  active = 'unobstructed_vf'
+
+  [unobstructed_vf]
+    type = UnobstructedPlanarViewFactor
+    boundary = 'top left right bottom'
+    execute_on = INITIAL
+  []
+
+  [vf_study]
+    type = ViewFactorRayStudy
+    execute_on = initial
+    boundary = 'left right bottom top'
+    face_order = TENTH
+    polar_quad_order = 80
+  []
+
+  [rt_vf]
+    type = RayTracingViewFactor
+    boundary = 'left right bottom top'
+    execute_on = INITIAL
+    normalize_view_factor = false
+    ray_study_name = vf_study
+  []
+[]
+
+[RayBCs]
+  active = ''
```

Comment:
For the purposes of the input file example in the doco, I would remove this and have `cli_args` in the tests file.

Reply:
done